sensor_array/analysis/sample_analysis.py

```diff
@@ -1123,3 +1124,29 @@ def read_kH_results(filename):
             temp_array.append(temp_row)
             full_array.extend(temp_array)
         return full_array
+
+
+def invert_matrix(array):
+    if np.shape(array)[0] == np.shape(array)[1]:
+        inv = np.linalg.inv(array)
+    else:
+        inv = np.linalg.pinv(array)
+
+    return inv
+
```

Comment:
I don't like the name here because it is too generic and sounds already like np.linalg.inv

Can you just use pinv instead of this method?